### PR TITLE
mimetype -> content_type

### PR DESCRIPTION
**Pull request by emord**
cc: @gcapalbo

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
-        "django<1.7",
+        "django<1.8",
         "jsonobject-couchdbkit",
-        "dimagi-utils>=1.3usec",
-        'django-soil==0.10.0',
-        'django-transfer==dev',
+        "dimagi-utils>=1.2.2",
+        'django-soil',
+        'django-transfer',
         "openpyxl",
```
Comment (gcapalbo) [inline setup.py]: do we need to pin django-soil or django-transfer anymore?

File: setup.py [hunk 2]
```diff
@@ -22,8 +22,7 @@
     packages=find_packages(exclude=['*.pyc']),
     include_package_data=True,
     dependency_links=[
-        'git+git://github.com/dimagi/dimagi-utils.git@24e1eaad37ac735ab9309253011f09835c9ee67e#egg=dimagi-utils-1.3usec',
-        'git+git://github.com/dimagi/django-soil.git@ad98c51dd95baec568c5f83af0807ef7965bf201#egg=django-soil-0.10.0',
-        'git+git://github.com/smartfile/django-transfer.git@6e0dc94c3341c358fca8eb2bf74e23aee3983ec4#egg=django-transfer-dev',
```
Comment (gcapalbo) [inline setup.py]: looks like you removed the link to django-transfer, just wanted to make sure that was intentional?

**Comment by czue** [comment]
(build failed)
```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.9/bin/django-admin.py", line 5, in <module>
    management.execute_from_command_line()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute
    django.setup()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/__init__.py", line 21, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/opt/python/2.7.9/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/travis/build/dimagi/couchexport/couchexport/models.py", line 8, in <module>
    from dimagi.ext.couchdbkit import Document, DictProperty,\
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/dimagi/ext/couchdbkit.py", line 2, in <module>
    from couchdbkit.ext.django.schema import *
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/ext/django/schema.py", line 24, in <module>
    from django.db.models.options import get_verbose_name
ImportError: cannot import name get_verbose_name
```

File: setup.py (hunk 1 above)
Comment (emord) [inline setup.py]: Yeah. it didn't pull down the correct versions unless they were pinned

File: setup.py (hunk 2 above)
Comment (emord) [inline setup.py]: It was intentional as it didn't make a difference for me when I ran these tests and it seemed better to be pulling down a real version vs pointing at a random commit. I looked at hq though and it's also pinned to this version, so it probably makes a difference there. Going to pin for now

**Pull request merged**